.github/workflows/phpunit.yml

```
@@ -0,0 +1,91 @@
+name: PHPUnit
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - stable*
+
+jobs:
+  php:
+    runs-on: ubuntu-latest
+
+    strategy:
+      # do not stop on another job's failure
+      fail-fast: false
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
+        databases: ['sqlite', 'mysql']
+        server-versions: ['master']
+  
+    name: php${{ matrix.php-versions }}/${{ matrix.databases }} on ${{ matrix.server-versions }}
+
+    services:
+      mariadb:
+        image: mariadb:latest
+        ports:
+          - 3306
```

Ah you need to expose the port, maybe the following works:

```suggestion
          - 3306:3306
```

---

tried as well.
Ubuntu is shipped with mysql service enabled, so you need to stop it, but I tried with a different port, still nothing